CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0
+
+* `DEPS`: update to `bpmn-js@11.1.0`
+* `DEPS`: update to `diagram-js@11.4.1`
```

Comment:
As `camunda-bpmn-js` is the entry point for other tools to integrate bpmn-js, the changelog should also contain noteworthy changes from the upstream libs. I see that we only explicitly mention changes made in camunda-bpmn-js.

Are there changes that impact the integration in the new bpmn-js or diagram-js versions? If there are only small changes in the new versions, I'm fine to release as-is

Reply:
Looks good to me, let's merge 👍